scripts/install.sh

```diff
@@ -132,7 +137,7 @@ Aborting now!
     ubuntu|debian)
         echo "# Installing pre-requisites..."
         $PRIVILEGED_EXECUTION "apt-get update"
-        $PRIVILEGED_EXECUTION "apt-get install -y gnupg apt-transport-https"
+        $PRIVILEGED_EXECUTION "apt-get install -y gnupg apt-transport-https curl"
```

Comment:
I think this will fail because curl is run even before this script.

Reply:
for debian, where?